packages/vitest-pool-workers/src/pool/index.ts

```diff
@@ -1248,3 +1263,24 @@ export default function (ctx: Vitest): ProcessPool {
 		},
 	};
 }
+
+/**
+ * Ensures that the specified compatibility feature is set correctly.
+ * @param flags The list of current flags.
+ * @param enableFlag The flag that enables the feature.
+ * @param disableFlag The flag that disables the feature.
+ */
+function ensureFlag(flags: string[], enableFlag: string, disableFlag: string) {
```

Comment:
As discussed offline, using a `Set` could simplify the handling of the flags in this PR.

I find it hard to figure out what that means:

```ts
	ensureFlag(
		runnerWorker.compatibilityFlags,
		"enable_nodejs_tty_module",
		"disable_nodejs_tty_module"
	);
```

ensure what flag (enable vs disable) is what (used vs not used)?

Maybe a wrapper class could help, constructed with an array, using a Set internally, having `ensureUsed(flag)` and `ensureNotUsed(flag)` and a `flags` getter returning an array

Reply:
I reworked the helper to make its purpose more clear: https://github.com/cloudflare/workers-sdk/pull/11047/commits/ab1f757c95f79f83395c6bda3780240e2920f659